commit 3937e91a0c8682fc50bcb170d3c1bef4a199a112 (refs/changes/12/2512/2)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-09-13T19:35:18+00:00 (2 years, 1 month ago)
    
    MANTA-3413 node-fash seems to intend add-data behavior it does not enable

diff --git a/bin/fash.js b/bin/fash.js
index ea04d9c..607cc17 100755
--- a/bin/fash.js
+++ b/bin/fash.js
@@ -210,7 +210,7 @@ Fash.prototype.do_deserialize_ring.help = (
 Fash.prototype.do_add_data = function (subcmd, opts, args, callback) {
     var self = this;
 
-    if (opts.help || args.length !== 0 || !opts.v || !opts.d || !opts.b) {
+    if (opts.help || args.length !== 0 || !opts.v || !opts.b) {
         this.do_help('help', {}, [subcmd], function (err) {
             return callback(err ? err : true);
         });
diff --git a/package.json b/package.json
index 8935c96..38bf673 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "dynamo",
     "hash ring"
   ],
-  "version": "2.4.0",
+  "version": "2.4.2",
   "repository": {
     "type": "git",
     "url": "https://github.com/yunong/node-fash.git"
